README.md

```diff
@@ -1,52 +1,39 @@
 # probe-rs
-[![crates.io](https://meritbadge.herokuapp.com/probe-rs)](https://crates.io/crates/probe-rs) [![documentation](https://docs.rs/probe-rs/badge.svg)](https://docs.rs/probe-rs) [![Actions Status](https://github.com/probe-rs/probe-rs/workflows/CI/badge.svg)](https://github.com/probe-rs/probe-rs/actions) [![chat](https://img.shields.io/badge/chat-probe--rs%3Amatrix.org-brightgreen)](https://matrix.to/#/!vhKMWjizPZBgKeknOo:matrix.org)
 
-A debugging toolset and library for debugging microchips on a separate host.
+a modern, embedded debugging toolkit,
+written in Rust
 
-## Motivation
+[![crates.io](https://meritbadge.herokuapp.com/probe-rs)](https://crates.io/crates/probe-rs) [![documentation](https://docs.rs/probe-rs/badge.svg)](https://docs.rs/probe-rs) [![Actions Status](https://github.com/probe-rs/probe-rs/workflows/CI/badge.svg)](https://github.com/probe-rs/probe-rs/actions) [![chat](https://img.shields.io/badge/chat-probe--rs%3Amatrix.org-brightgreen)](https://matrix.to/#/#probe-rs:matrix.org)
 
 The goal of this library is to provide a toolset to interact with a variety of embedded MCUs and debug probes.
-For starters, ARM cores will be supported through use of the CoreSight protocol.
-If there is high demand and more contributors, it is intended to add support for other architectures.
 
 Similar projects like OpenOCD, PyOCD, Segger Toolset, ST Tooling, etc. exist.
 They all implement the GDB protocol and their own protocol on top of it to enable GDB to communicate with the debug probe.
-This is not standardized and also little bit unstable sometimes. For every tool the commands are different and so on.
 
-This project gets rid of the GDB layer and provides a direct interface to the debug probe,
-which then enables other software, for example [VisualStudio](https://code.visualstudio.com/blogs/2018/08/07/debug-adapter-protocol-website) to use it's debug functionality.
+This project gets rid of the GDB layer (even tho we provide one you can use until [Microsoft DAP](https://code.visualstudio.com/blogs/2018/08/07/debug-adapter-protocol-website) support is fully implemented) and provides a direct interface to the debug probe,
```

Comment:
I would remove the sentence in brackets, and maybe add a separate section for the `gdb-server` and `vscode` projects?